Formula/emacs.rb

```diff
@@ -94,3 +96,12 @@ def plist
     assert_equal "4", shell_output("#{bin}/emacs --batch --eval=\"(print (+ 2 2))\"").strip
   end
 end
+
+__END__
+diff --git a/lisp/site-load.el b/lisp/site-load.el
+new file mode 100644
+index 0000000..408a62d
+--- /dev/null
++++ b/lisp/site-load.el
+@@ -0,0 +1 @@
++(setq exec-path (delete nil (mapcar (lambda (elt) (unless (string-match-p "Homebrew/shims" elt) elt)) exec-path)))
```

Comment:
Could this be done as something like
```ruby
("list/site-load.el").write <~EOS
+(setq exec-path (delete nil (mapcar (lambda (elt) (unless (string-match-p "Homebrew/shims" elt) elt)) exec-path)))
EOS
```

Reply:
Yeah, I didn't really know what the preferred method was here, was just trying to fix this: https://github.com/Homebrew/homebrew-core/runs/1067888651 I'll push a fix.

Reply:
@SMillerDev brew audit doesn't seem to like this:

```
    ("lisp/site-load.el").write <~EOS
    (setq exec-path (delete nil
                     (mapcar
                      (lambda (elt)
                       (unless (string-match-p "Homebrew/shims" elt) elt))
                      exec-path)))
    EOS
```

gives:

```
brew audit --strict  ~/Developer/Ruby/homebrew-core/Formula/emacs.rb
Error: emacs: /Users/camdennarzt/Developer/Ruby/homebrew-core/Formula/emacs.rb:60: syntax error, unexpected ( arg, expecting do or '{' or '('
    (setq exec-path (delete nil
                    ^
/Users/camdennarzt/Developer/Ruby/homebrew-core/Formula/emacs.rb:61: syntax error, unexpected '(', expecting ')'
                     (mapcar
                     ^
/Users/camdennarzt/Developer/Ruby/homebrew-core/Formula/emacs.rb:63: syntax error, unexpected tSTRING_BEG, expecting do or '{' or '('
...      (unless (string-match-p "Homebrew/shims" elt) elt))
...                              ^
/Users/camdennarzt/Developer/Ruby/homebrew-core/Formula/emacs.rb:63: syntax error, unexpected tIDENTIFIER, expecting ')'
...ng-match-p "Homebrew/shims" elt) elt))
...                            ^~~
/Users/camdennarzt/Developer/Ruby/homebrew-core/Formula/emacs.rb:64: syntax error, unexpected ')', expecting end
...                    exec-path)))
...                              ^
```

Reply:
done